bitnami/elasticsearch/ci/ct-values.yaml

```diff
@@ -1,6 +1,9 @@
 master:
-  replicas: 1
+  replicaCount: 3
+  masterOnly: false
 data:
-  replicas: 1
+  replicaCount: 0
 coordinating:
-  replicas: 1
+  replicaCount: 0
+ingest:
+  replicaCount: 0
```

Comment:
Not less consuming, but with 1 + 1 + 1 you have no High Availability really. I personally will always recommend High availability, because, in a 1-1-1 scenario: If you lose your master, the cluster won't operate; if you lose the 1 data node, the cluster will be Red; if you lose the 1 coordinating node, cluster won't be reachable. While having a 3 multi-role cluster you will be tolerant to losing 1 node, cluster upgrades will be rolled smoothly...

The same thing happens with the current default 2-2-2-2, but I didn't want to change that yet to not surprise anybody with a topology change during this already big major.

Dedicated nodes benefits really come in handy when having a heavy ingestion cluster, or simply more than 6 nodes holding data, so you want those data nodes to be 100% dedicated to r/w operations, and master operations start consuming more resources (so you move it to a dedicated-master to not make those operations in a data node).